css.md

```diff
@@ -14,15 +14,56 @@ CSS should not be handwritten. It should be preprocessed through [Stylus](http:/
 
 ## Property ordering
 
-Properties should be in this general order:
+SOP orders CSS properties by importance. This approach prioritizes and groups different /types/ of CSS rules, in order to present the /most significant/ styles first.
 
-* Display
-* Positioning
-* Box model
-* Colors and Typography
-* Other (transitions, animations, translations)
+Maintaining this order makes our stylesheets more scannable, and more instructive. Mapping CSS properties according to the significance of their effect shortens the search for any CSS property that drastically changes an element’s position, stacking context, or size.
 
-This ordering help clarify how an element relates with other models first, then its more isolated style attributes.
+Mixins, an `@extend`, or variables which output CSS (such as `font-specs(x, x, x)` from `sop-styl`) should be placed at the top of its relevant group.
```

Comment:
This might be adjusted if i can get property lookups to work the way i want them to avoid dup properties.